backend/app/gtrans.py

```
@@ -623,18 +623,21 @@ async def transcribe_audio(audio_path: str, status_queue: asyncio.Queue, transcr
+    Provider-agnostic: supports Groq, Ollama, MiniMax, Alibaba Qwen, etc.
+    Configure via CLOUD_API_BASE_URL and CLOUD_API_KEY environment variables.
     Currently NOT IMPLEMENTED.
     """
-    error_msg = "Groq transcription pathway is defined but not yet implemented."
+    error_msg = "Cloud transcription pathway is defined but not yet implemented."
     logger.error(error_msg)
     console.print(f"[bold red]Error: {error_msg}[/bold red]")
     # Send error message to frontend via queue
     await status_queue.put(json.dumps({"type": "error", "content": error_msg}))
-    logger.info(f"QUEUE PUT (Error): Groq not implemented")
+    logger.info(f"QUEUE PUT (Error): Cloud transcription not implemented")
```

Ruff (0.16.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)